### ISSUE TITLE
STORY: As a browser extension developer, I want a clear and validated specification of the tutorial functionality so that I can implement and test a new tutorial that can be tested with end users.

### ISSUE THREAD

**Issue by Cybernide** [Shared-Reality-Lab/IMAGE-browser#301]
- [x] Decide on approach to tutorial given implementable options. See: https://github.com/Shared-Reality-Lab/IMAGE-UX/issues/49
- [x] Create wireframe/mockup of approach
- [x] Create wireframe/mockup of button interfaces for photos. See: https://github.com/Shared-Reality-Lab/IMAGE-browser/issues/281

**Comment by Cybernide** [comment]
Smaller buttons
https://mydraft.cc/cgdk359term4uu9bqujg

**Comment by jeffbl** [comment]
Moving to backlog since not complete.

**Comment by jeffbl** [comment]
@Cybernide is this still a valid item? If so, can you summarize what needs to be done yet?
